### ISSUE TITLE
"With the release of netlify dev this package should be used anymore."

### ISSUE THREAD

**Issue by DavidSabine** [porfirioribeiro/gatsby-plugin-netlify-functions#10]
In the readme, is that supposed to read: "With the release of netlify dev this package should *NOT*..."

**Comment by DavidSabine** [comment]
Also, if one uses `netlify dev` instead, would you recommend still using `gatsby-plugin-netlify-functions` for any other purpose -- such as to define a "functionsSrc" and "functionsOutput".  (It's really handy that webpack does the work to transpile and move code from one folder to the other.)  In my local environment right now, there doesn't appear to be any problem leaving `gatsby-plugin-netlify-functions` in my environment.

**Comment by DavidSabine** [comment]
Update:

Yes, with the release of `netlify dev`, this package should not be used anymore.

Also: if one uses netlify dev instead, then there's no need for gatsby-plugin-netlify-functions.  The only features that netlify dev doesn't include (which gatsby-plugin-netlify-functions *does* include) is transpilation and hot-reloading of the functions.  Transpilation can be achieved with `netlify lambda` -- but hot-reloading of the functions is still an open question (for me).

I made a video this morning to demonstrate how I've refactored my environment to use netlify dev (within a yarn workspace setting) to replace gatsby-plugin-netlify-functions. (https://www.youtube.com/watch?v=-bPTpVOLxfM)
